opam-ci-check/bin/main.ml

```diff
@@ -155,29 +125,34 @@ let lint_cmd =
   in
   Cmd.v info term
 
-let list_cmd =
-  let doc = "List the revdeps for a package" in
+let no_cache =
+  let info = Arg.info [ "no-cache" ] ~doc:"Don't use the docker cache" in
+  Arg.value (Arg.flag info)
+
+let build_test_cmd =
+  let doc = "Build and test a package" in
   let term =
     Term.(
-      const show_revdeps $ pkg_term $ local_opam_repo_term
-      $ no_transitive_revdeps)
+      const build_run_spec $ no_cache $ const BuildTest $ pkg_term
+      $ local_opam_repo_term)
```

Comment:
I think it would be helpful to have a CLI that allowed just dumping the obuilder spec (optionally translated to a docker file). Do you have any ideas on what a sensible CLI for that might look like?

Reply:
I think it could be done as a flag to the `build` command to only dump the spec and not actually run the build step?